vmdk_plugin/drivers/vmdk/vmdkops/esx_vmdkcmd.go

```diff
@@ -111,26 +111,33 @@ func (vmdkCmd EsxVmdkCmd) Run(cmd string, name string, opts map[string]string) (
 			var errno syscall.Errno
 			errno = err.(syscall.Errno)
 			msg = fmt.Sprintf("Run '%s' failed: %v (errno=%d).", cmd, err, int(errno))
-
-			// Still below maximum number of retries, log and continue
+			str := C.GoString(ans.errBufP)
+			if str != "" {
```

Comment:
why do you need to compare ? would not  simply `msg +=  C.GoString(ans.errBuf)` be exactly the same as L114-L117 combined ? I think you can even add it directly to L113